src/Cake.Parallel/Cake.Parallel.Module.csproj

```diff
@@ -1,13 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.6</TargetFrameworks>
```

Comment:
Slightly skeptical on the need to switch to `netstandard2.0` here, only because I have an older project using this module. I'll need to get back to you on this one before I pull this in.